client/src/elm/Pages/PatientRecord/View.elm

```diff
@@ -96,15 +104,310 @@ viewContentForChild language currentDate zscores childId child isChw initiator d
+                        db
+                        SetActivePage
+                )
+                entries
+                |> Maybe.Extra.values
+                |> List.sortWith sortTuplesByDateDesc
+                |> List.map Tuple.second
+    in
+    div [ class "pane acute-illness" ]
+        [ viewPaneHeading language <| Translate.PatientRecordFilter FilterAcuteIllness
+        , div [ class "pane-content" ] <|
+            entriesHeading
+                :: viewEntries language daignosisEntries
+        ]
+
+
+viewAntenatalPane :
+    Language
+    -> NominalDate
+    -> PersonId
+    -> List ( IndividualEncounterParticipantId, IndividualEncounterParticipant )
+    -> ModelIndexedDb
+    -> Html Msg
+viewAntenatalPane language currentDate personId pregnancies db =
+    let
+        ( activePregnancies, completedPregnancies ) =
+            List.partition (Tuple.second >> isPregnancyActive currentDate) pregnancies
+
+        entriesHeading =
+            div [ class "heading antenatal" ]
+                [ div [ class "date-start" ] [ text <| translate language Translate.PregnancyStart ]
+                , div [ class "status" ] [ text <| translate language Translate.StatusLabel ]
+                , div [ class "date-end" ] [ text <| translate language Translate.PregnancyConclusion ]
+                , div [ class "see-more" ] [ text <| translate language Translate.SeeMore ]
+                ]
+
+        entries =
+            List.map (\( participantId, data ) -> ( ( participantId, StatusOngoing ), data )) activePregnancies
+                ++ List.map (\( participantId, data ) -> ( ( participantId, StatusResolved ), data )) completedPregnancies
+
+        daignosisEntries =
+            List.map (viewAntenatalEntry language currentDate personId db) entries
+                |> Maybe.Extra.values
+                |> List.sortWith sortTuplesByDateDesc
+                |> List.map Tuple.second
+    in
+    div [ class "pane antenatal" ]
+        [ viewPaneHeading language <| Translate.PatientRecordFilter FilterAntenatal
+        , div [ class "pane-content" ] <|
+            entriesHeading
+                :: viewEntries language daignosisEntries
+        ]
+
+
+viewAntenatalEntry :
+    Language
+    -> NominalDate
+    -> PersonId
+    -> ModelIndexedDb
+    -> ( ( IndividualEncounterParticipantId, PaneEntryStatus ), IndividualEncounterParticipant )
+    -> Maybe ( NominalDate, Html Msg )
+viewAntenatalEntry language currentDate personId db ( ( participantId, status ), data ) =
+    let
+        encounters =
+            getPrenatalEncountersForParticipant db participantId
+
+        maybeLastEncounterId =
+            List.head encounters
+                |> Maybe.map Tuple.first
+    in
+    Maybe.map
+        (\lastEncounterId ->
+            let
+                startDate =
+                    Maybe.map (Date.add Days -280 >> formatDDMMYYYY) data.eddDate
```

Comment:
@anvmn Maybe worth to add a comment of what the -280 is. I guess this is 40 weeks (in days) of pregnancy, right?

Reply:
@mariano-dagostino implemented at 5438007